samples/GalleryApp/MainPage.xaml

```diff
@@ -39,7 +39,7 @@
                                Padding="10, 60, 0, 0"
                                TextWrapping="Wrap"/>
                 </Grid>
-                <Button Height="400" Width="400" Opacity="0" Click="NavigateToSkillPage"/>
+                <Button Height="400" Width="400" Opacity="0.5" Click="NavigateToSkillPage" Style="{StaticResource ButtonRevealStyle}"/>
```

Comment:
>ButtonRevealStyle [](start = 114, length = 17)

where is this defined?

Reply:
It's a keyword from the UWP style documentation: [https://docs.microsoft.com/en-us/windows/uwp/design/style/reveal](https://docs.microsoft.com/en-us/windows/uwp/design/style/reveal)


---
In reply to: [296978865](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/35#discussion_r296978865) [](ancestors = 296978865)